packages/oauth/package.js

```diff
@@ -37,7 +37,7 @@ Package.onUse(api => {
 });
 
 Npm.depends({
-  'body-parser': '1.19.0',
+  'body-parser': '1.20.3',
```

Comment:
If #13323 as whole is not acceptable, then I think we can cherry-pick the part that just removes this dependency as we have this in Express.

Reply:
It's not that it's unacceptable, but it involves different changes and context than the contributions we merged for 3.2. We need time to analyze and test the affected parts, including any side effects. Especially since some areas on those changes may not have full test coverage. Deferring some changes helps us focus our efforts.

That PR could be a good candidate for the first beta of 3.3. As with 3.2, this would give time for testing, including our final verifications before the official release.

Reply:
Btw, I have a list of the PRs you attach in release milestones, including that one, so we can review and decide what to include in future releases. We plan to do this for each release based on the challenges on those, the testing capacity on us and the community to test before official. Keep flagging issues in milestones, some areas are hard to track, and those warnings help a lot.

Reply:
Yes, feel free to move the issues/PRs to their appropriate release milestones. We should probably jump on a call/stream sometime and do a few together and talked about this so that I can better triage.